scripts/cmake/vcpkg_build_qmake.cmake

```
@@ -63,23 +63,25 @@ function(vcpkg_build_qmake)
     set(targets_release "${arg_RELEASE_TARGETS}")
 
     foreach(build_type IN ITEMS debug release)
```

```suggestion
    if(NOT DEFINED VCPKG_BUILD_TYPE)
        set(items debug release)
    else()
        set(items release)
    endif()
    foreach(build_type IN ITEMS ${items})
```
Note: release only builds are not really release builds but more like single configurations builds.

---

also remove all other changes